webapp/channels/src/components/advanced_create_comment/advanced_create_comment.test.tsx

```diff
@@ -418,19 +455,20 @@ describe('components/AdvancedCreateComment', () => {
 
     test('should open PostDeletedModal when message is submitted to deleted root', () => {
         const onUpdateCommentDraft = jest.fn();
-        const draft = {
+        const draft: any = {
             message: 'Test message',
             uploadsInProgress: [1, 2, 3],
             fileInfos: [{id: '1', name: 'aaa', create_at: 100}, {id: '2', name: 'bbb', create_at: 200}],
         };
-        const props = {...baseProps, onUpdateCommentDraft, draft};
+        const props: any = {...baseProps, onUpdateCommentDraft, draft};
 
-        const wrapper = shallow(
+        const wrapper = shallow<AdvancedCreateComment>(
             <AdvancedCreateComment {...props}/>,
         );
 
         wrapper.setProps({rootDeleted: true});
-        wrapper.instance().handleSubmit({preventDefault: jest.fn()});
+        preventDefaultObject.preventDefault = jest.fn();
```

Comment:
You don't actually need to modify `preventDefault` in each of these tests since the mocked function is reset between tests, so you can remove the lines where you're doing this